src/platforms/common/configuration/options.mdx

```diff
@@ -372,6 +372,12 @@ _(New in version 6.0.0)_
 
 </ConfigKey>
 
+<ConfigKey name="attach-screenshot" supported={["iOS"]}>
+
+Takes a screenshot of the application when an error or exception is manually captured and includes it as an attachment.
```

Comment:
It doesn't capture on crashes?

Reply:
No. We didn't agree on trying to run obj-c during crashs. It's possible, but not recommended.

Reply:
We should document that here https://docs.sentry.io/platforms/android/enriching-events/screenshots/